LocustScripts/commcarehq-referrals-search-for-beds.py

```diff
@@ -0,0 +1,325 @@
+        self.client.headers.update(headers)
+
+        with self.client.post(f"{self.user.formplayer_host}/{command}/", json=json, name=name,
+                              catch_response=True) as response:
+            data = response.json()
+            # logging.info("data-->" + str(data))
+            if "notification" in data and data["notification"]:
+                if data["notification"]["type"] == "error":
+                    logging.info("ERROR::-" + data["notification"]["message"] + ": With json" + str(json))
+                    response.failure("exception error--" + data["notification"]["message"])
+            if "exception" in data:
+                logging.info("ERROR::exception error--" + data['exception'])
+                logging.info("ERROR::user-info::" + self.user.username + "::" + self.user.login_as)
+                response.failure("exception error--" + data['exception'])
+            elif checkKey and checkKey not in data:
+                logging.info("error::" + checkKey + " not in data")
+                response.failure("ERROR::" + checkKey + " not in data")
+            elif checkKey and checkLen:
+                if len(data[checkKey]) != checkLen:
+                    logging.info("ERROR::len(data['" + checkKey + "']) != " + checkLen)
+                    response.failure("error::len(data['" + checkKey + "']) != " + checkLen)
+            elif checkKey and checkValue:
+                if data[checkKey] != checkValue:
+                    logging.info("ERROR::data['" + checkKey + "'] != " + checkValue)
+                    response.failure("error::data['" + checkKey + "'] != " + checkValue)
+        return response.json()
+
+
+class LoginCommCareHQWithUniqueUsers(HttpUser):
+    tasks = [WorkloadModelSteps]
+
+    formplayer_host = "/formplayer"
+    project = 'bha-referrals-perf'  # str(os.environ.get("project"))
+    domain_user_credential_force = str(os.environ.get("user_credential"))
+    app_config_force = str(os.environ.get("app_config"))
+    wait_time_force = "test"
+
+    if wait_time_force == "test":
+        wait_time = between(5, 10)
+
+    else:
+        wait_time = between(45, 90)
+
+    with open("project-config/" + project + "/config.yaml") as f:
+        config = yaml.safe_load(f)
+        host = config['host']
+        domain = config['domain']
+        app_id = config['app_id']
+        if domain_user_credential_force != "None":
+            domain_user_credential = "project-config/" + project + "/" + domain_user_credential_force
+        else:
+            domain_user_credential = config['domain_user_credential']
+        if app_config_force != "None":
+            app_config = "project-config/" + project + "/" + app_config_force
+        else:
+            app_config = config['app_config']
+
+    # get domain user credential and app config info
+    with open(domain_user_credential) as json_file:
+        data = json.load(json_file)
+        data_user = data['user']
```

Comment:
Really not great to have this static code as part of the user class initialization. 

I would recommend using an event listener for the `init` event to do global setup e.g. https://github.com/dimagi/dimagi-qa/pull/324/files#diff-5b0844709ffdc4635e4332a75709404e1d32e9e4c89944069e61e8b1e559d15bR27

Reply:
Curious why that's not preferable, but changed here https://github.com/dimagi/dimagi-qa/pull/323/commits/914caa18bdbf814af846e455c62f261b21d179b7